scripts/ci/docker-compose/backend-mssql.yml

```diff
@@ -38,3 +40,14 @@ services:
       timeout: 10s
       retries: 10
     restart: always
+  mssqlsetup:
+    image: mcr.microsoft.com/mssql/server:${MSSQL_VERSION}
+    depends_on:
+      mssql:
+        condition: service_healthy
+    entrypoint:
+      - bash
+      - -c
+      - opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P Airflow123 -i /mssql_create_airflow_db.sql | true
```

Comment:
Should we have double `|` here?

Reply:
Yep. I noticed and fixed it at about the same time you did